src/Tribe/Repository.php

```diff
@@ -1361,10 +1361,6 @@ public function get_ids() {
 			 */
 			return array();
 		}
-
-		$query->set( 'fields', 'ids' );
-
```

Comment:
@mitogh why is this removed? We're in the `get_ids` method so it seems getting IDs only is the right thing to do.

Reply:
My bad just rolled back thanks.